tests/test_fixity.py

```diff
@@ -291,7 +312,7 @@ def test_scan_handles_exceptions_if_no_scan_attempted(_get, environment):
     ],
     ids=["Success", "Fail", "Did not run"],
 )
-def test_scan_message(status, error_message):
+def test_scan_message(status: mock.Mock, error_message: mock.Mock) -> None:
```

Comment:
These parameters are not `Mock` objects. See https://docs.pytest.org/en/latest/example/parametrize.html and the reference of the `parametrize` mark https://docs.pytest.org/en/latest/reference/reference.html#pytest.Metafunc.parametrize

The `argvalues` in this test a couple lines above are tuples of `(bool, str)`. That's what the test will receive.